Cargo.toml

```diff
@@ -85,7 +87,7 @@ members = [
     "opencensus-proto",
     "opentelemetry-proto",
     "spiffe-proto",
-    "tools",
+    "tools", "linkerd/http/override-authority",
```

Comment:
format nit: sort and put on it's own line

Reply:
thank you for spotting this!

this is moved onto its own sorted line in the amended commit 5f9c146